### DISCUSSION TITLE
Next 14.2.x is showing different build output for functions

### DISCUSSION THREAD

**Discussion by fungilation** [vercel/next.js#64758] [Help] [answered]
### Summary

on Vercel builds dashboard. Previously on 14.1.4:

![image](https://github.com/vercel/next.js/assets/3803466/a694add9-57a3-44b8-b445-3b3b11e619d3)

now on 14.2.1:

![image](https://github.com/vercel/next.js/assets/3803466/32fe775f-444e-456f-8855-7012a5468208)

I haven't changed the functions that were on "Edge Runtime", so did Vercel builds just changed to no longer differentiate between Edge Runtime and "Dynamic"?

I also see the symbols changed from before λ, ℇ to ƒ for all dynamic fns. So is this missing differentiation a Vercel dashboard issue?

**Comment by jlison** [comment]
Could this be the cause: https://twitter.com/leeerob/status/1780705942734331983 ?

**Reply by fungilation** [reply]
ok, replied to his tweet and copied here. BAD move. Let me choose when I actually want the Edge "runtime", I don't care about the geo these functions

> NO @leerob. This is destroying functions I built that works with the Edge "runtime". I want close to 0 cold start, and I want cpu only usage billing.
> 
> I don't want Node.js for that for certain use cases
> 
> I guess I have to move to Deno or Cloudflare now

**Reply by leerob** [reply]
This is not related to this change in the build output (that haven't been changes to the edge runtime in Next.js, it still exists).

**Comment by leerob** [comment, marked as answer]
We changed the output format to use `ƒ` for all dynamic functions. If you are wanting to see the infrastructure outputs in the Vercel dashboard, you can look at the deployment summary. 

https://vercel.com/docs/deployments/overview#deployment-summary

**Reply by fungilation** [reply]
ok, so just dashboard reporting change, not the Edge runtime functions been removed. Fine by me
